### PR TITLE
Initial File Provider Service Update

### PR DESCRIPTION
**Pull request by awrigh11**
Initial Commit for the File Provider Service

### CODE REVIEW

File: service/src/Main.java
Comment (awrigh11) [inline service/src/Main.java:1]: These Main classes are unnessary

File: api/pom.xml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    </parent>
+    <artifactId>file-provider-service-api</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <url>https://github.com/awrigh11/datawave-file-provider-service-awrigh11</url>
+    <scm>
+        <connection>scm:git:https://github.com/awrigh11/datawave-file-provider-service-awrigh11.git</connection>
+        <developerConnection>scm:git:git@github.com:awrigh11/datawave-file-provider-service-awrigh11.git</developerConnection>
+        <tag>HEAD</tag>
+        <url>https://github.com/awrigh11/datawave-file-provider-service-awrigh11</url>
+    </scm>
```
Comment (foster33) [inline api/pom.xml:18]:
```suggestion
    <url>https://github.com/NationalSecurityAgency/datawave-file-provider-service</url>
    <scm>
        <connection>scm:git:https://github.com/NationalSecurityAgency/datawave-file-provider-service.git</connection>
        <developerConnection>scm:git:git@github.com:NationalSecurityAgency/datawave-file-provider-service.git</developerConnection>
        <tag>HEAD</tag>
        <url>https://github.com/NationalSecurityAgency/datawave-file-provider-service</url>
    </scm>
```
Should this be changed? This is the only pom I noticed was different.

Comment (awrigh11) [reply api/pom.xml:18]: Yes I will fix

File: api/pom.xml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        <relativePath>../../../microservice-parent/pom.xml</relativePath>
+    </parent>
+    <artifactId>file-provider-service-api</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <url>https://github.com/awrigh11/datawave-file-provider-service-awrigh11</url>
+    <scm>
+        <connection>scm:git:https://github.com/awrigh11/datawave-file-provider-service-awrigh11.git</connection>
+        <developerConnection>scm:git:git@github.com:awrigh11/datawave-file-provider-service-awrigh11.git</developerConnection>
+        <tag>HEAD</tag>
+        <url>https://github.com/awrigh11/datawave-file-provider-service-awrigh11</url>
```
Comment (dtspence) [inline api/pom.xml]: Is it possible to update the connection information here to be generic?

Comment (awrigh11) [reply api/pom.xml]: Does the change I made to this file in the latest commit address your concern?

Comment (dtspence) [reply api/pom.xml]: Yes, thank you.

**Review by dtspence — APPROVED** [review]

**Review by jjonesy16 — APPROVED** [review]

**Pull request merged**